docs/source/introduction/configuring.rst

```diff
@@ -66,9 +66,12 @@ contain the following::
    [Paths]
```

Comment:
We should have a note warning people about the change of behavior in Python 3.8 and how to modify their code to get things working again. We can then explain how to persist this option in the config file.

Reply:
Do you have a style of 'comment' to follow? This one feels more squishy...

Reply:
I took a stab at updating the FAQ and the Configuring the backend section